### PR TITLE
Update inspec for os[:family] change in Train

### PR DESCRIPTION
**Pull request by stevendanna**
Signed-off-by: Steven Danna steve@chef.io

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,6 +8,13 @@ if Gem::Version.new(RUBY_VERSION) <= Gem::Version.new('1.9.3')
   gem 'net-ssh', '~> 2.9'
 end
 
+
+#
+# This has to be merged someone concurrently with a a version bump in
+# train to avoid breaking users.
+#
+gem 'train', github: 'chef/train'
```
Comment (chris-rock) [inline Gemfile:16]: We need to remove it, once we release

File: lib/resources/pip.rb [hunk 2]
```diff
@@ -57,9 +57,7 @@ def to_s
     def pip_cmd
       # Pip is not on the default path for Windows, therefore we do some logic
       # to find the binary on Windows
-      family = inspec.os[:family]
-      case family
-      when 'windows'
+      if inspec.os.windows?
```
Comment (chris-rock) [inline lib/resources/pip.rb:60]: +1 makes it simpler!

File: lib/resources/postgres.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@ class Postgres < Inspec.resource(1)
 
     attr_reader :service, :data_dir, :conf_dir, :conf_path
     def initialize
-      case inspec.os[:family]
+      case inspec.os[:name]
```
Comment (chris-rock) [inline lib/resources/postgres.rb]: should we use os.debian? here?

File: lib/resources/windows_feature.rb [hunk 4]
```diff
@@ -42,7 +42,7 @@ def initialize(feature)
       @cache = nil
 
       # verify that this resource is only supported on Windows
-      return skip_resource 'The `windows_feature` resource is not supported on your OS.' if inspec.os[:family] != 'windows'
+      return skip_resource 'The `windows_feature` resource is not supported on your OS.' if !inspec.os.windows?
```
Comment (chris-rock) [inline lib/resources/windows_feature.rb:45]: +1

File: test/helper.rb [hunk 5]
```diff
@@ -38,25 +38,25 @@
...
+    osx104:     { name: 'mac_os_x',family: 'darwin', release: '10.10.4', arch: nil },
+    ubuntu1204: { name: 'ubuntu', family: 'debian', release: '12.04', arch: 'x86_64' },
+    ubuntu1404: { name: 'ubuntu', family: 'debian', release: '14.04', arch: 'x86_64' },
+    ubuntu1504: { name: 'ubuntu', family: 'debian', release: '15.04', arch: 'x86_64' },
+    windows:    { name: 'windows', family: 'windows', release: '6.2.9200', arch: 'x86_64' },
+    wrlinux:    { name: 'wrlinux', family: 'redhat', release: '7.0(3)I2(2)', arch: 'x86_64' },
+    solaris11:  { name: "solaris", family: 'solaris', release: '11', arch: 'i386'},
+    solaris10:  { name: "solaris", family: 'solaris', release: '10', arch: 'i386'},
+    hpux:       { name: 'hpux', family: 'hpux', release: 'B.11.31', arch: 'ia64'},
+    undefined:  { name: nil, family: nil, release: nil, arch: nil },
```
Comment (chris-rock) [inline test/helper.rb:59]: awesome! Thanks @stevendanna

**Comment by stevendanna** [comment]
The Travis changes have the nice affect of giving you at-a-glance information about which tests are which:

screen shot 2016-08-04 at 1 34 33 pm

**Comment by chris-rock** [comment]
I like the travis changes. Thanks for their introduction @stevendanna !

**Comment by chris-rock** [comment]
Great work @stevendanna

**Comment by stevendanna** [comment]
Fixes #855 (we hope)

**Pull request merged**